### ISSUE TITLE
No doc for displayArrayKey and 

### ISSUE THREAD

**Issue by FrankYufeiYang** [mac-s-g/react-json-view#426]
I think you forgot to add doc for displayArrayKey

**Comment by shashiirk** [comment]
Yes. I got this typescript error when I tried to use that property...

`Property 'displayArrayKey' does not exist on type 'IntrinsicAttributes & ReactJsonViewProps & { children?: ReactNode; }'`

Maintainers, please resolve this!
